File: Python/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
    def __init__(self, id=None, nama=None, harga=None, stok=None, jenis=None, bahan=None, warna=None, untuk=None, size=None, merk=None):
        super().__init__(id, nama, harga, stok, jenis, bahan, warna)
        self.untuk = untuk
        self.size = size
        self.merk = merk

    # setter dan getter

    # setter untuk
    def set_untuk(self, untuk):
        self.untuk = untuk

    # setter size
    def set_size(self, size):
        self.size = size

    # setter merk
    def set_merk(self, merk):
        self.merk = merk

    # getter untuk
    def get_untuk(self):
        return self.untuk

    # getter size
    def get_size(self):
        return self.size

    # getter merk
    def get_merk(self):
        return self.merk

File: Java/Baju.java
public class Baju extends Aksesoris{
    private String untuk;
    private String size;
    private String merk;

    public Baju(){
    }

    public Baju(int id, String nama, double harga, int stok, String jenis, String bahan, String warna, String untuk, String size, String merk){
        super(id, nama, harga, stok, jenis, bahan, warna);
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    public void setUntuk(String untuk){
        this.untuk = untuk;
    }

    public void setSize(String size){
        this.size = size;
    }

    public void setMerk(String merk){
        this.merk = merk;
    }

    public String getUntuk(){
        return untuk;
    }

    public String getSize(){
        return size;
    }

    public String getMerk(){
        return merk;
    }
    
}